frontend/src/metabase/home/homepage/components/HomeModelCard/HomeModelCard.styled.tsx

```diff
@@ -16,4 +16,5 @@ export const CardTitle = styled(Ellipsified)`
   font-size: 1rem;
   font-weight: bold;
   margin-left: 1rem;
+  max-width: 300px;
```

Comment:
I believe it should be `max-width: 100%` here with 

```
${breakpointMinMedium} {
  max-width: 50%;
}
```

or something like that

Reply:
Thank you @ranquild 

Makes sense to me.

I've refined the parameters.

There's a Loom video in the PR description.

Separate thing, but looks like we could make the padding around the group of these links smaller in mobile, to give more space to the text in the links.